data/json/mapgen/lab/lab_modular/lab_nests_modular/lab_nested_maintenance.json

```diff
@@ -0,0 +1,508 @@
+          { "item": "tailorbooks", "chance": 60, "repeat": [ 0, 1 ] },
+          { "item": "SUS_tailoring_fasteners", "chance": 30, "repeat": [ 2, 6 ] }
+        ],
+        "w": [ { "item": "SUS_tailoring_materials", "chance": 70, "repeat": [ 1, 3 ] } ]
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "A nested map for maintenance areas.",
+    "nested_mapgen_id": "lab_maintenance_3x3_EW_open",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "rows": [
+        "qqq",
+        "   ",
+        "QQQ"
+      ],
+      "palettes": [ "lab_common_palette" ],
+      "items": {
+        "q": [ { "item": "tools_common", "chance": 70, "repeat": [ 1, 3 ] } ],
+        "Q": [ { "item": "tools_electronics", "chance": 70, "repeat": [ 1, 3 ] } ]
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "A nested map for maintenance areas.",
+    "nested_mapgen_id": "lab_maintenance_3x3_EW_open",
+    "object": {
+      "mapgensize": [ 3, 3 ],
+      "rows": [
+        "IIY",
+        " A ",
+        "www"
+      ],
+      "palettes": [ "lab_common_palette" ],
+      "items": {
+        "I": [
+          { "item": "SUS_tailoring_tool_drawer", "chance": 60, "repeat": [ 0, 1 ] },
+          { "item": "tailorbooks", "chance": 60, "repeat": [ 0, 1 ] },
+          { "item": "SUS_tailoring_fasteners", "chance": 30, "repeat": [ 2, 6 ] }
+        ],
+        "w": [ { "item": "SUS_tailoring_materials", "chance": 70, "repeat": [ 1, 3 ] } ]
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "A nested map for maintenance areas.",
+    "nested_mapgen_id": "lab_maintenance_6x6_open",
+    "object": {
+      "mapgensize": [ 6, 6 ],
+      "rows": [
+        "vv   g",
+        "   5 g",
+        " A₸5  ",
+        "  Z5  ",
+        "   5 g",
+        "uu   g"
+      ],
+      "palettes": [ "lab_common_palette", "lab_maintenance_palette" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "A nested map for maintenance tunnel content.",
+    "nested_mapgen_id": "lab_maintenance_5x5_room_NE",
+    "object": {
+      "mapgensize": [ 5, 5 ],
+      "rows": [
+        "||;||",
+        "|;;;|",
+        "|;^;;",
+        "|;;;|",
+        "|||||"
+      ],
+      "palettes": [ "lab_common_palette", "lab_maintenance_palette" ],
+      "place_loot": [
+        { "item": "cheese_hard", "x": 2, "y": [ 1, 3 ], "chance": 100 },
+        { "item": "blanket", "x": 1, "y": 3, "chance": 100 },
+        { "item": "flask_glass", "x": 1, "y": 3, "chance": 100 },
+        { "item": "iv_mutagen_rat", "x": [ 1, 3 ], "y": [ 1, 3 ], "chance": 100 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "A nested map for maintenance tunnel content.",
+    "nested_mapgen_id": "lab_maintenance_5x5_room_NE",
+    "object": {
+      "mapgensize": [ 5, 5 ],
+      "rows": [
+        "||;||",
+        "|;;;|",
+        "|;^;;",
```

Comment:
actually 2 different symbols `ʌ` and `^`  Good point though and I'll look for a less similar solution (and push a different PR for the fix in case it effects other maps).